README.md

```diff
@@ -4,33 +4,23 @@ This is the source code/files for the ehcache.org website.  The files in this re
 ### Instructions for building/previewing
 
 * On Mac OS X, install Xcode command line tools by executing: `xcode-select --install`
-
-* Install Jekyll if you have not - follow instructions on the Jekyll home page (after first installing Ruby)
-[http://jekyllrb.com/](http://jekyllrb.com/)
-
-* Run `gem install bundle`
-
+* Install the toolchain
+  * (Recommended) Use [rvm](https://rvm.io) for handling Ruby locally, following instructions are designed for that.
+  * Install Ruby 2.2.1 - `rvm install 2.2.1`
+  * Create a new gemset in that ruby version - `rvm gemset create <name>`
+  * Make sure you are using the version and gemset - `rvm use 2.2.1@<name>`
+  * Import the gems found in default.gems - `rvm gemset import`
 * Clone this repository to your local system (if you're going to contribute content, fork it first, and clone that)
-* cd into the "ehcache.github.io" directory
-* "git checkout ehcache.org" to switch to this branch
-
-* Install dependencies with: `bundle install`
-
+  * Create a branch to contribute content
 * To generate the *full* site including Ehcache 3 docs, you need to do the following:
-  * within the root your clone of this repository, the contents of ehcache3 repository needs to be linked/or copied as "_eh3"
-  * within the root your clone of this repository, the contents of ehcache3 repository's docs/src/docs/asciidoc/user folder needs to be linked/or copied as "documentation/3.0"
-
-* To generate and view (locally serve) the site "bundle exec jekyll serve -w"   ( then point your browser at http://localhost:4000" )
-
-* To simply generate the site "jekyll build"
+  * within the root your clone of this repository, the contents of ehcache3 repository needs to be linked/or copied as `_eh3`
+  * within the root your clone of this repository, the contents of ehcache3 repository's docs/src/docs/asciidoc/user folder needs to be linked/or copied as `documentation/3.0`
+* To generate and view (locally serve) the site `jekyll serve` then point your browser at `http://localhost:4000`
```

Comment:
Good point - the error comes from the fact that jekyll ends up trying to watch twice the same folder.